lib/LaTeXML/Package/tikz.sty.ltxml

```diff
@@ -21,6 +21,10 @@ use strict;
 use warnings;
 use LaTeXML::Package;
 
+# TODO: What do we need to do to get the native emulation working?
+DefMacro('\pgfmathresult',   '0.0');
+DefMacro('\tikz@align@temp', '\pgfmathresult');
+
```

Comment:
Weird. It seems that the *loading* of `tikz` is already using these macros, but before it has defined them itself.

Reply:
I think they are defined during the load, but conditionally, and we will only have those definitions if our internal emulation is perfect.